src/rev_ai/apiclient.py

```diff
@@ -44,9 +45,11 @@ def __init__(self, access_token):
 
     def submit_job_url(
             self,
-            media_url,
+            source_url,
+            source_auth=None,
```

Comment:
@amikofalvy A lot of my confusion is around this, where we have a media_url parameter that's required and positional while nothing else is, and we want to add the source config option, which is not required, but also replaces the need for the media url. 

Can we make media_url a named parameter without breaking implementations? Or do we need to break it anyway to make this change?  Can hop on a zoom call if you want.

Reply:
I added a default option for media_url and added source_config at the end, so if anyone is relying on order of args rather than named args it won't break anything.